libs/aws/odc/aws/queue.py

```diff
@@ -83,6 +83,39 @@ def get_queue(queue_name: str):
     return queue
 
 
+def list_queues(region: Optional[str] = None):
+    """
+    Return a list of queues which the user is allowed to see
+    """
+    sqs = boto3.resource("sqs")
+    return list(sqs.queues.all())
+
+
+def get_queue_attributes(queue_name: str, attribute: Optional[str] = None) -> dict:
```

Comment:
I think this should accept a queue, not a string.

Reply:
I'm following what other functions already do such as :
[get_queue](https://github.com/opendatacube/odc-tools/blob/ad937a7aca4af5519ac03a034ff2a5cd97499637/libs/aws/odc/aws/queue.py#L81)

if I receive a queue as a parameter, the user must have to use **get_queue** before calling my function and will have to inform the queue name anyways.

Reply:
Look at `get_messages`